CHARTER.md

```diff
@@ -24,9 +24,9 @@ This is so we don't have to block the portable API on filling in a lot of missin
 
 ## Membership
 
-**Shepherds:** @hsivonen, @KodrAus, @Lokathor
-**Team Liason:** @KodrAus
-**Members:** @BurntSushi, @calebzulawski, @hsivonen, @KodrAus, @Lokathor, @workingjubilee
+* **Shepherds:** @hsivonen, @KodrAus, @Lokathor
```

Comment:
```suggestion
* **Shepherds:** @calebzulawski, @Lokathor, @workingjubilee
```

Reply:
This is exactly the list that was already there but just based on our initial meeting I think this is a more accurate reflection of the project shepherds 🙂